### PR TITLE
adding separate test for Power options buttons

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================
__Extending__  Power Control tests to check available power actions from  "on/off" because now many test are failing due to inconsistent list of available actions

{{pytest: cfme/tests/cloud/test_instance_power_control.py "-k test_start or test_stop or test_power_options_from_on or test_power_options_from_off" --long-running --use-provider azure}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 45c61cb905b8bc289feec6ec93e789df5e688708:

`cfme/tests/cloud/test_instance_power_control.py`:
- :red_circle: Line 9:1: **F401** *'BZ' imported but unused*
- :red_circle: Line 378:5: **N802** *function name should be lowercase*
- :red_circle: Line 390:5: **N802** *function name should be lowercase*

Please, rectify these issues :smirk: .

Commit inspection report:

**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit bde69b6647305c6fa1d4a370fbb233ec67a80d70:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit fdafc57269c07756b1b90ed7ff14776bb35c4d6c:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by lkhomenk** [comment]
thi PR is to avoid :

**Comment by lkhomenk** [comment]
the best run one can get is http://10.16.4.32/trackerbot/pr/run/8085 - on 5.7 new functions are expected to fail due to open BZ https://bugzilla.redhat.com/show_bug.cgi?id=1385587 
i've also excluded "azure" from test as it's failing too much on Setup, which is not related to this PR

**Comment by lkhomenk** [comment]
after merging https://github.com/ManageIQ/wrapanapi/pull/82 - azure may be added to testrun also

File: cfme/tests/cloud/test_instance_power_control.py [hunk 1]
```diff
@@ -25,7 +24,7 @@ def pytest_generate_tests(metafunc):
 def testing_instance(request, setup_provider, provider):
     """ Fixture to provision instance on the provider
     """
-    instance = Instance.factory(random_vm_name('pwr-ctrl'), provider)
+    instance = Instance.factory(random_vm_name('pwr-c'), provider)
```
Comment (RonnyPfannschmidt) [inline cfme/tests/cloud/test_instance_power_control.py]: why the shortening?

Comment (lkhomenk) [inline cfme/tests/cloud/test_instance_power_control.py]: 1st version was made right after 1st version of name generator - default  length less than 40 symbols.
and after name generator started giving 15symbols long  names by default - last part was cut anyway. so just to meet defaults

Comment (RonnyPfannschmidt) [inline cfme/tests/cloud/test_instance_power_control.py]: oh i see, i feel that sideeffect is unintentional and should be canged in future

@psav do we want to do something about that (my personal preference would be to have more meaningful prefixes at the cost of randomization)

Comment (psav) [inline cfme/tests/cloud/test_instance_power_control.py]: Perhaps so, but I'm OK with this right now

**Comment by lkhomenk** [comment]
http://10.16.4.32/trackerbot/pr/run/8117 - failed where it supposed to - 5.7 has an open BZ for missing Delete, +1 failed teardown which is not part of this PR

**Pull request merged**